presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -517,6 +517,31 @@ public void testBinaryFunctions()
         // from_base64, to_base64.
         assertQuery("SELECT from_base64(to_base64(cast(comment as varbinary))) FROM orders");
 
+        // from_big_endian_32, to_big_endian_32.
+        assertQuery("SELECT from_big_endian_32(to_big_endian_32(null))");
+        assertQuery("SELECT from_big_endian_32(to_big_endian_32(0))");
+        assertQuery("SELECT from_big_endian_32(to_big_endian_32(1))");
+        assertQuery("SELECT from_big_endian_32(to_big_endian_32(-1))");
+        assertQuery("SELECT from_big_endian_32(to_big_endian_32(12345678))");
```

Comment:
Should add a couple of one-way tests also.
```
assertQuery("SELECT to_big_endian_32(12345678)");
assertQuery("SELECT from_big_endian_32(12345678)");
```

Reply:
Absolutely, I will add those tests.